examples/evs-csi-plugin/kubernetes/resize/pod.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: resize-nginx
+spec:
+  containers:
+    - image: nginx
+      imagePullPolicy: IfNotPresent
+      name: nginx
+      ports:
+        - containerPort: 80
+          protocol: TCP
+      volumeMounts:
+        - mountPath: /var/lib/www/html
+          name: csi-data-resize
+  volumes:
+    - name: csi-data-resize
+      persistentVolumeClaim:
+        claimName: pvc-resize-demo
+        readOnly: false
```

Comment:
pls add a black line

Reply:
done, has changed in branch "feat-documents-yaml"